src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,64 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    public static void getStatistic(String fromFileName, String toFileName) {
```

Comment:
split your logic into 3 private methods as it is described in the common mistakes:
* `String[] readFromFile(String fileName)`
* `String[] formReport(String[] data)`
* `writeToFile(String[] data, String fileName)`
feel free to use just string instead of array